libunwind/src/cet_unwind.h

```diff
@@ -39,3 +40,5 @@ extern void *__libunwind_cet_get_registers(unw_cursor_t *);
 extern void *__libunwind_cet_get_jump_target(void);
```

Comment:
What is the problem with exposing these two declarations, even if they're not defined?

Reply:
there are no registers. so what would you expect for cet_get_jump_target? there is no setjmp/longjmp